tox.ini

```diff
@@ -12,25 +13,25 @@ deps =
     black
 skip_install = True
 commands =
-    black --check src/ssort tests setup.py
+    black --check .
 
 [testenv:isort]
 deps =
     isort
 skip_install = True
 commands =
-    isort --check-only src/ssort tests setup.py
+    isort --check-only .
 
 [testenv:ssort]
 commands =
-    ssort --check --diff src/ssort tests setup.py
+    ssort --check --diff src/ssort tests
```

Comment:
We need to fix this.  Tracking in https://github.com/bwhmather/ssort/issues/72.  Surprised pylint and pyflakes haven't implemented something similar.